docs/advanced-topics/reconnecting-mid-game.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I don't see any link to our auto reconnect code in boss room?

Reply:
(currently pointing to develop since this feature is not in a released version of BR yet. When the release happens, we should update this link to point to main instead)